### PR TITLE
Product Page

### PR DESCRIPTION
**Pull request by andrew032011**
### Summary 

This pull request is the first step towards implementing the product page. Changes:
- Implement a products card component with a hover-over effect
- Products is an array of all the data for each product and render the products list
- Single vertical line is present between each product card
- Responsiveness shadowing effect when hovering over a card

- [x] implemented Products page and product card component

Figma: https://www.figma.com/file/ICgD6k2eL9TNVUaCk8E97h/FA-22-Handoff?node-id=0%3A1&t=OntIKNu0dsCkUSR2-0

Notion: https://www.notion.so/cornelldti/Implement-Products-page-ce94c807282246e3a69fc8eb231934ec

### Test Plan 

https://user-images.githubusercontent.com/59291082/201491062-db7a56cf-a850-4ba5-953d-4b61926d4de4.mov

### Notes 

- Increased margin-bottom of footer size by a bit. I needed to increase the size to avoid some blank white space at the bottom of the products page

### CODE REVIEW

File: dti-website-redesign/src/pages/products/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+  },
+  {
+    imageURL: '/static/products/cornellgo.png',
+    siteURL: '/products/cornellgo',
+    year: 'Fall 2021',
+    name: 'Cornell Go',
+    description:
+      'Helping students find professors and classes that work best for them, CU Reviews began in 2017 as a database of student reviews and testimonies.'
+  }
+];
```
Comment (henry-li-06) [inline dti-website-redesign/src/pages/products/index.tsx:78]: This data should probably be put into a `.json` file somewhere to make it easier for people to update product info

**Review by henry-li-06 — COMMENTED** [review]
Is responsiveness to screen width not in the scope of this PR?

**Comment by JacksonStaniec** [comment]
> Is responsiveness to screen width not in the scope of this PR?

The only guidelines in this handoff are about large screen sizes:
image
image
Smaller screens and mobile is not in scope.

File: dti-website-redesign/src/pages/products/index.tsx (hunk 1 above)
Comment (andrew032011) [reply dti-website-redesign/src/pages/products/index.tsx]: Agreed, done.

**Comment by andrew032011** [comment]
> > Is responsiveness to screen width not in the scope of this PR?
> 
> The only guidelines in this handoff are about large screen sizes: image image Smaller screens and mobile is not in scope. @andrew032011 i don't see any max-width in the css for the product cards tho.

Whoops, forgot about that. Also incorporated some other details from the Figma into the CSS that I had forgotten.

**Comment by andrew032011** [comment]
Ready for another pass.

**Review by JacksonStaniec — APPROVED** [review]
This looks great! You've demonstrated that you're as capable with the frontend as you are with the backend!

**Pull request merged**